test/helpers/cilium.go

```diff
@@ -333,6 +350,93 @@ func (c *Cilium) ReportFailed(commands ...string) {
 		fmt.Fprint(wr, res.Output())
 	}
 	fmt.Fprint(wr, "StackTrace Ends\n")
+	c.ReportDump()
+}
+
+// ReportDump runs a variety of commands and writes the results to
+//testResultsPath
+func (c *Cilium) ReportDump() {
+	reportCmds := map[string]string{}
+
+	for cmd, log := range ciliumCLICommands {
+		reportCmds[fmt.Sprintf("sudo %s", cmd)] = log
```

Comment:
You are still running commands with `sudo` that do not need to be ran with `sudo`. If you need commands to be ran with sudo, add `sudo` to the command itself in `ciliumCLICommands`, not here.